src/types/index.d.ts

```diff
@@ -20,15 +21,20 @@ type AtLeast<T, K extends keyof T> = Partial<T> & Pick<T, K>
 type RoutePath = import("../server/routes").SupportedRoutes
 type RouteInfo = Record<string, string | (() => JSX.Element | null)>
 
-type KratosFlowData = { registrationData?: SelfServiceRegistrationFlow }
+type KratosFlowRegistrationData = { registrationData?: SelfServiceRegistrationFlow }
```

Comment:
My idea here was to define:
```
type KratosFlowData = {
  registrationData?: SelfServiceRegistrationFlow
  loginData?: SelfServiceLoginFlow
  (...)
}
```

If you use a union type instead `type KratosFlowData = KratosFlowRegistrationData | KratosFlowLoginData` I guess you should make it consistent and remove the `?` in `KratosFlowRegistrationData` or?